### ISSUE TITLE
-Wparentheses in mm/memcontrol.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#629]
Reported-by: kernelci.org bot <bot@kernelci.org>
```
   mm/memcontrol.c:1160:17: warning: using the result of an assignment as a condition without parentheses [-Wparentheses]
    mm/memcontrol.c:1160:17: note: place parentheses around the assignment to silence this warning
    1 warning generated.
```

**Comment by nathanchance** [comment]
This has a pending fix: https://lore.kernel.org/lkml/1564580753-17531-1-git-send-email-cai@lca.pw/

**Comment by nathanchance** [comment]
Fixed as of `next-20190807`: https://git.kernel.org/next/linux-next/c/09e9529550df815068532cd96b31ef64794b6c43
